File: Easy/JavaScript/628. Maximum Product of Three Numbers.js
var maximumProduct = function(nums) {
    nums.sort((a, b) => a - b);
    let twoNegatives = nums[0] * nums[1] * nums[nums.length - 1];
    let noNegatives = nums[nums.length - 1] * nums[nums.length - 2] * nums[nums.length - 3];
    return Math.max(noNegatives, twoNegatives);
};

File: Easy/Java/628. Maximum Product of Three Numbers.java
class Solution {
    public int maximumProduct(int[] nums) {
        Arrays.sort(nums);
        int twoNegatives = nums[0] * nums[1] * nums[nums.length - 1];
        int noNegatives = nums[nums.length - 1] * nums[nums.length - 2] * nums[nums.length - 3];
        return Math.max(noNegatives, twoNegatives);
    }
}
